projects/templates/home.html

```diff
@@ -52,22 +52,34 @@
           <div class="container-fluid">
             <a class="navbar-brand" href="#">Navbar</a>
             <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-              <span class="navbar-toggler-icon">
-                <form class="d-flex" role="search">
-                  <div class="input-group mb-3">
-                    <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
-                    <input type="text" class="form-control" placeholder="Search the blog" aria-label="Search the blog" aria-describedby="button-addon2">
-                    <button type="button" class="btn btn-success">
-                      <i class="fa fa-search"></i>
-                    </button>
-                  </div>
-                </form>
-              </span>
+              <span class="navbar-toggler-icon"></span>
             </button>
-
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
```

Comment:
**important note** all the menu content - except logo- should be inside this div that has the ` id="navbarSupportedContent"`
if we want the menu to be responsive